packages/riipen-ui/src/components/Divider.test.jsx

```diff
@@ -0,0 +1,98 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Divider from "./Divider";
+
+describe("<Divider>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Divider />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Divider().type.defaultProps;
+
+      const wrapper = mount(<Divider />);
+
+      const component = wrapper.find("Divider");
+      expect(component.props().variant).toEqual(defaultProps.variant);
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const classes = ["riipen", "riipen-divider"];
+
+      const wrapper = mount(<Divider />);
+
+      expect(
+        wrapper
+          .find("Divider")
+          .props()
+          .classes.sort()
+      ).toEqual(classes.sort());
+    });
```

Comment:
Not sure this should be bested inside the `default props` describe block. We aren't really testing the default props here.